docker/runtime/dotnetcore-2.0/Controllers/FunctionController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using kubeless_netcore_runtime.Util;
+
+namespace kubeless_netcore_runtime.Controllers
+{
+    //[Produces("application/json")]
+    [Route("/")]
+    public class FunctionController : Controller
+    {
+
+        [HttpPost]
+        public object Post([FromBody]object data)
+        {
+            try
+            {
+                return CompileAndExecuteFunction(data);
+            }
+            catch (Exception ex)
+            {
+                return ex.Message;
+            }
+        }
+
+        [HttpGet]
+        public string Get()
+        {
+            return "NotImplemented";

```

Comment:
The other runtimes support GET and POST, I think we can leave it up to the function to error if there is no post data passed to it.

Reply:
Yes, you are right. I will throw a NotImplementedException.

Reply:
Implemented on current commit